docs/physics/intro/light_and_matter.rst

```diff
@@ -0,0 +1,115 @@
+
+Note that these are sometimes called bound-bound interactions, since the electron starts bound to an atom and ends bound to the same atom.
+
+Ionization (bound-free interactions)
+------------------------------------
+
+The next type of interaction is called ionization. This is when a photon interacts with an electron and gives the electron enough energy to escape from the atom, as shown in the diagram below. Unlike line interactions, the final energy of the electron is not restricted to discrete energy levels since the electron is not bound to an atom, the difference in energy :math:`E_f-E_i` of the electron can be anything above the ionization energy :math:`E_\mathrm{ion}`, or the minimum energy it takes to ionize the atom. This energy can be seen ???. So, an ionizing photon can have any frequency greater than or equal to :math:`\frac{E_\mathrm{ion}}{h}`. Note that this is called ionization because the final atom is missing an electron and is thus an ion.
+
+.. figure:: images/ionization.png
+
+    Image from https://idahoaskascientist.com/2016/01/08/why-are-gamma-rays-more-harmful-than-radio-waves/.
+
+Ionization is also called a bound-free interaction, since the electron starts bound to an atom and ends up not bound to an atom (i.e. free).
+
+Note that just like with de-excitation, ionization can be "reversed" through **recombination**. This is where any free electron (not necessarily one that was ionized off the particular atom) becomes bound to an atom (recombines with the atom). The electron's energy is lowered to one of the energy levels of the atom, and a photon is released with an energy equal to the amount of energy the electron loses in the recombination process.
+
+Electron Scattering
+-------------------
+
+The final type of interaction is electron scattering. This is when a photon collides with a free electron, which then, like any two particles colliding, sends the electron and the light off in different directions in accordance with the `conservation of momentum <https://www.physicsclassroom.com/class/momentum/Lesson-2/Momentum-Conservation-Principle>`_. This is shown in the diagram below. If the electron is moving far slower than the speed of light, this is called `Thompson scattering <https://en.wikipedia.org/wiki/Thomson_scattering>`_, and the wavelength/frequency/energy of the photon remains the same before and after the collision. If the electron is moving close to the speed of light, `relativity <https://en.wikipedia.org/wiki/Special_relativity>`_ kicks in, and we have `Compton scattering <https://en.wikipedia.org/wiki/Compton_scattering>`_ in which the wavelength/frequency/energy of the photon will change after the collision. In Thompson scattering, photons of any frequency have the same likelihood of scattering with an electron. In Compton scattering, while any photon of any frequency can scatter, the likelihood of scattering depends on the energy/frequency of the photon.
```

Comment:
Thompson -> Thomson

Reply:
Don't use / between quantities like energy and frequency, it could be misinterpreted as a division (last sentence)